src/msr_data_arch.cpp

```diff
@@ -242,6 +242,15 @@ namespace geopm
                     "behavior":  "variable",
                     "writeable": false
                 },
+                "LIMIT_CPUID_MAXVAL": {
+                    "begin_bit": 22,
+                    "end_bit":   22,
+                    "function":  "scale",
+                    "units":     "none",
+                    "scalar":    1.0,
+                    "behavior":  "variable",
```

Comment:
constant?

Reply:
The LIMIT_CPUID_MAXVAL is listed as R/W in the SDM Vol4 Table 2-2. IA-32 Architectural MSRs.  Others appear to be the same.